src/WebJobs.Script/Description/PowerShell/PowerShellFunctionDescriptorProvider.cs

```diff
@@ -33,7 +33,10 @@ public override bool TryCreate(FunctionMetadata functionMetadata, out FunctionDe
 
         protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath, BindingMetadata triggerMetadata, FunctionMetadata functionMetadata, Collection<FunctionBinding> inputBindings, Collection<FunctionBinding> outputBindings)
         {
-            return new PowerShellFunctionInvoker(Host, functionMetadata, inputBindings, outputBindings);
+            throw new NotSupportedException();
+
+            // TODO: FACAVAL
+            // return new PowerShellFunctionInvoker(Host, functionMetadata, inputBindings, outputBindings);
```

Comment:
Can we reenable PowerShell?

Reply:
Had a quick chat with @paulbatum. All of the experimental languages will be dropped from 2.0 for the time being. We have an issue tracking that (#1744).

We'll likely use the worker/language extensibility model as we bring them back in the future.

Reply:
That issue doesn't say ALL languages will be dropped, but calls out Python/PHP. I understand that we also need to make improvements to Powershell, so we're adding that to the list. Can you update the issue to also include Powershell then?

To confirm, in this PR you're only dropping support for Powershell/Python/PHP then?